app.rb

```diff
@@ -90,5 +91,5 @@ def verify_signature(payload_body)
 end
 
 def redmine_issue_repos
-  ENV['REDMINE_ISSUE_REQUIRED_REPOS'].to_s.split.map { |r| Regexp.new("\\A#{r}\\z") }
+  YAML.load_file('config/redmine_issue_required_repos.yaml')
```

Comment:
Indeed, fine by me - you'll just need to expand the wildcard that's currently in the YAML file, and hopefully we won't have a need for any more.